lib/review-apps.unit.test.js

```diff
@@ -1,39 +1,110 @@
 require('isomorphic-fetch');
 const nock = require('nock');
+const pRetry = require('p-retry');
 
-jest.mock('./github-api');
 const reviewApps = require('./review-apps');
+
 const BASE_HEROKU_API_URL = 'https://api.heroku.com';
 
+jest.mock('./github-api');
+
 describe('review-apps', () => {
+
 	let nockScope;
 	let reviewAppsNockScope;
+
 	beforeAll(() => {
+
 		nockScope = nock(BASE_HEROKU_API_URL)
 			.matchHeader('accept', 'application/vnd.heroku+json; version=3');
+
 		reviewAppsNockScope = nock(BASE_HEROKU_API_URL)
 			.matchHeader('accept', 'application/vnd.heroku+json; version=3.review-apps');
+
 		jest.mock('shellpromise', () => a => a, { virtual: true });
 
 		// WARNING: Disable error logs, to clean up test output
 		// Re-enable here if need be
 		jest.spyOn(console, 'error').mockImplementation().mockName('console.error');
+
 	});
 
 	afterAll(() => {
+
 		jest.unmock('shellpromise');
 		jest.unmock('./github-api');
+
 	});
 
 	afterEach(() => {
+
 		nock.cleanAll();
 		jest.resetAllMocks();
+
+	});
+
+	describe('runCheckForBuildSuccessStatus', () => {
+
+		const { runCheckForBuildSuccessStatus } = reviewApps;
+
+		it('returns without throwing an error if awaiting the review app build does not error', async () => {
```

Comment:
Can this test be removed? The response is mocked so in essence its testing `nock` works correctly

Reply:
The reasons I added it were:
- There is a corresponding test for when an error is thrown to ensure that it in that scenario it throws a specific type of error (`pRetry.AbortError`), so seemed logical to have a the flip-side of that scenario (i.e. no error thrown). I wanted to achieve coverage of scenarios as comprehensive as possible so that they were all accounted for (which inevitably results in a few tests with minimal value such as this one).
- There is (admittedly minimal) value in knowing that it simply returns once it has finished its async business without returning a specific value (though admittedly this is not explicit in the test).
- This function also relies on the success of the other functions called before it encounters `nock`: `repeatedCheckForBuildSuccessStatus()`, `checkForBuildSuccessStatus()`, `getAppBuildWithCommit()`, `getBuilds()`, `getBuildsUrl()`, which is inadvertently also being tested.

But I take your point and am happy to remove if you still think it does not provide sufficient value?

Reply:
I see your thinking behind. Lets not remove it but maybe we can tweak the assertion to checking that it doesnt error?